src/3d/qgs3dmapsettings.h

```diff
@@ -355,6 +356,18 @@ class _3D_EXPORT Qgs3DMapSettings : public QObject
      */
     double outputDpi() const { return mDpi; }
 
+    /**
+     * Returns the mesh symbol used to render mesh terrain
+     * \since QGIS 3.12
+     */
+    QgsMesh3DSymbol meshterrainSymbol() const;
+
+    /**
+     * Sets the mesh symbol used to render mesh terrain
+     * \since QGIS 3.12
+     */
+    void setMeshterrainSymbol( const QgsMesh3DSymbol &meshterrainSymbol );
+
```

Comment:
These methods should not be needed anymore if the mesh terrain symbol is stored within mesh terrain generator class, right?

Reply:
right, it is coming!